src/main/resources/templates/bootstrap/bootstrap_nav_header.html

```diff
@@ -31,7 +31,7 @@
           Course Offering
         </a>
       </li>
-      <li th:if="${isAdmin}" class="nav-item ">
+      <li th:if="${isTutor || isAdmin}" class="nav-item ">
```

Comment:
Recall as mentioned earlier, only admins should be able to adjust tutor assignments